src/main/java/de/hhu/propra/sharingplatform/dao/ImageService.java

```diff
@@ -0,0 +1,44 @@
+package de.hhu.propra.sharingplatform.dao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.util.MimeType;
+import org.springframework.util.ResourceUtils;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.annotation.Resource;
+import javax.servlet.http.HttpServletRequest;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URL;
+
+@Component
+public class ImageService {
+
+    @Autowired
+    private HttpServletRequest request;
+
+    public void store(MultipartFile file, String filename) {
+        if (file.getContentType().equals("image/png") || file.getContentType().equals("image/jpeg")) {
+            try {
+                File out = ResourceUtils.getFile("src/main/resources/static/images/" + filename);
```

Comment:
This one is a little hacky... Better Idears are welcome!!! What happens when we put it in a jar? location will change...

Reply:
I guess there is a ways to set the path in the manifest file.